src/Components/Articles.jsx

```diff
@@ -1,8 +1,30 @@
+import { useState } from "react"
+import axios from "axios"
+import ArticleCard from "./ArticleCard"
+
 function Articles() {
+    const [articles, setArticles] = useState([])
+    function handleApi() {
```

Comment:
can we extract the api handler so that the function is concerned only with the rendering of the component?